src/store/lookup.ts

```diff
@@ -44,7 +60,41 @@ async function lookupCard(browser: Browser, store: Store, page: Page, link: Link
 	const givenWaitFor = store.waitUntil ? store.waitUntil : 'networkidle0';
 	const response: Response | null = await page.goto(link.url, {waitUntil: givenWaitFor});
 
-	if (await lookupCardInStock(store, page)) {
+	if (!response) {
+		Logger.debug(Print.noResponse(link, store, true));
+	}
+
+	let backoff = storeBackoff[store.name];
+
+	if (!backoff) {
+		backoff = {count: 0, time: Config.browser.minBackoff};
+		storeBackoff[store.name] = backoff;
+	}
+
+	if (response?.status() === 403) {
+		Logger.warn(Print.backoff(link, store, backoff.time, true));
+		await delay(backoff.time);
+		backoff.count++;
+		backoff.time = Math.min(backoff.time * 2, Config.browser.maxBackoff);
```

Comment:
> 403 (Forbidden) responses are retried with exponential backoff to allow IP rate limits to reset

This keeps it from going to infinite then, correct?

Reply:
Yes, up to a default of one hour.